src/styles/Home.module.css

```diff
@@ -1,18 +1,18 @@
-.listings_container {
-    padding: 1em;
-    display: grid;
+.listingsContainer {
```

Comment:
@Sheyitofunmi after changing these CSS properties, did you change them also in the files where they are referenced? If you do not do this, those parts of the app will lose their styling